great_expectations/expectations/metrics/column_aggregate_metrics/column_nonnull_count.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from great_expectations.compatibility.pyspark import functions as F
+from great_expectations.compatibility.sqlalchemy import sqlalchemy as sa
+from great_expectations.execution_engine import (
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.metrics.column_aggregate_metric_provider import (
+    ColumnAggregateMetricProvider,
+    column_aggregate_partial,
+    column_aggregate_value,
+)
+
+
+class ColumnNonNullCount(ColumnAggregateMetricProvider):
+    metric_name = "column.nonnull_count"
+    value_keys = ()
```

Comment:
`value_keys` should be inherited from the superclass so you should be able to remove this line.